qiskit_nature/second_q/operators/electronic_integrals.py

```diff
@@ -0,0 +1,482 @@
+        """
+        return ElectronicIntegrals(
+            self.alpha.conjugate(),
+            self.beta.conjugate(),
+            self.mixed.conjugate(),
+        )
+
+    def transpose(self) -> ElectronicIntegrals:
+        """Transpose of ``ElectronicIntegrals``.
+
+        Returns:
+            The transpose of the ``ElectronicIntegrals``.
+        """
+        return ElectronicIntegrals(
+            self.alpha.transpose(),
+            self.beta.transpose(),
+            self.mixed.transpose(),
+        )
+
+    @classmethod
+    def einsum(
+        cls,
+        einsum_map: dict[str, tuple[str, ...]],
+        *operands: ElectronicIntegrals,
+    ) -> ElectronicIntegrals:
+        """Exposes the :meth:`qiskit_nature.second_q.operators.PolynomialTensor.einsum` method.
+
+        This behaves identical to the einsum implementation of the ``PolynomialTensor``, applied to
+        the :attr:`alpha`, :attr:`beta`, and :attr:`mixed` attributes of the provided
+        ``ElectronicIntegrals`` operands.
+
+        This method is special, because it handles the scenario in which any operand has a non-empty
+        :attr:`beta` attribute, in which case the empty-beta attributes of any other operands will
+        be filled with :attr:`alpha` attributes of those operands.
+        The same applies to the :attr:`mixed` attributes.
+
+        Args:
+            einsum_map: a dictionary, mapping from :meth:`numpy.einsum` subscripts to a tuple of
+                strings. These strings correspond to the keys of matrices to be extracted from the
+                provided ``ElectronicIntegrals`` operands. The last string in this tuple indicates the
+                key under which to store the result in the returned ``ElectronicIntegrals``.
+            operands: a sequence of ``ElectronicIntegrals`` instances on which to operate.
+
+        Returns:
+            A new ``PolynomialTensor``.
+        """
+        alpha = PolynomialTensor.einsum(einsum_map, *(op.alpha for op in operands))
+
+        beta: PolynomialTensor = None
+        if any(not op.beta.is_empty() for op in operands):
+            # If any beta-entry is non-empty, we have to perform this computation.
+            # Empty tensors will be populated with their alpha-terms automatically.
+            beta = PolynomialTensor.einsum(
+                einsum_map,
+                *(op.alpha if op.beta.is_empty() else op.beta for op in operands),
+            )
+
+        mixed: PolynomialTensor = None
+        if all(not op.mixed.is_empty() for op in operands):
+            # We can only perform this operation, when all mixed tensors are non-empty.
+            mixed = PolynomialTensor.einsum(einsum_map, *(op.mixed for op in operands))
+        return ElectronicIntegrals(alpha, beta, mixed)
+
+    # pylint: disable=invalid-name
+    @classmethod
+    def from_raw_integrals(
+        cls,
+        h1: np.ndarray,
+        h2: np.ndarray | None = None,
```

Comment:
Ah the lack of these suffixes is likely an artifact from when these were overloaded to also allow SO-basis matrices..
Now that they no longer do that (because then a user would construct a `PolynomialTensor` directly (something I should add to the documentation)) I can add those suffixes to be consistent :+1: